lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -83,14 +84,17 @@ def load_from_data(self, model, course_structure, scores_client, submissions_sco
         """
         for block in model.visible_blocks.blocks:
             persisted_values = {'weight': block.weight, 'possible': block.max_score}
+            locator = BlockUsageLocator.from_string(block.locator)
             self._compute_block_score(
                 User.objects.get(id=model.user_id),
-                block.locator,
+                locator,
```

Comment:
We discussed making the BlockRecord object store BULs instead of strings, and only stringifying during serialization.  Is that still where we want to go with this?  It seems like that would also solve this issue.